src/main/java/racingcar/domain/TotalRacingResults.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TotalRacingResults {
+	private List<RacingResults> totalResults;
+
+	public TotalRacingResults(List<RacingResults> totalResults) {
+		if (totalResults == null) {
+			totalResults = new ArrayList<>();
```

Comment:
```suggestion
			this.totalResults = Optional.ofNullable(totalResults)
				.orElse(Collections.emptyList());
```

외부에서 전달한 객체인 totalResults 파라미터가 null이었는데 list로 셋팅해버리면 위험합니다
외부의 값이 갑자기 상태가 바뀌면 어느 객체가 영향을 미쳤는지 코드 하나하나 분석해야되는 일이 생길 수 있습니다
차라리 null로 두는게 낫습니다
this.totalResults를 초기화하는게 낫겠네요